README.md

```diff
@@ -78,37 +78,71 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Let's say you want to train your `LinearRegression` model by your data `X`, `y`. After splitting the data into train/test segments you would have `X_train` and `y_train` as your training data. So you train your model.
 ```pycon
->>> from sklearn import datasets
->>> from pymilo import Export, Import
->>> from sklearn.linear_model import LinearRegression
->>> import os
->>> X, Y = datasets.load_diabetes(return_X_y=True)
->>> threshold = 20
->>> X_train, X_test = X[:-threshold], X[-threshold:]
->>> Y_train, Y_test = Y[:-threshold], Y[-threshold:]
 >>> model = LinearRegression()
->>> #### Train the model using the training sets
 >>> model.fit(X_train, Y_train)
 ```
-### Save model 
+
+### Save model - Serialize
+Now, let's say you want to have a transparent `.json` file from your trained model. Using PyMilo you can easily have than.
 ```pycon
```

Comment:
- same point for the 'let's say' part.

- also the "transparent `.json` file" part misled me that maybe other formats we have. other than `.json` or also another option than being `transparent`?

- the last word, "than"?